pyquil/paulis.py

```diff
@@ -675,13 +678,13 @@ def get_qubits(self):
         """
         return list(set().union(*[term.get_qubits() for term in self.terms]))
 
-    def simplify(self):
+    def simplify(self) -> 'PauliSum':
         """
         Simplifies the sum of Pauli operators according to Pauli algebra rules.
         """
         return simplify_pauli_sum(self)
 
-    def get_programs(self):
+    def get_programs(self) -> Tuple[List[Program], List[NumberT]]:
```

Comment:
sequence

Reply:
The first element of the tuple is `programs = [term.program for term in self.terms]` which is an explicit list. The second element of the tuple is `coefficients = np.array([term.coefficient for term in self.terms])` which is actually an ndarray.

Reply:
imo if the second element is an `ndarray`, then the type hint should reflect the actual type say `np.ndarray`, not `List`

Reply:
my bad, got a little trigger happy with the sequences